src/main/java/org/gridsuite/study/server/StudyConstants.java

```diff
@@ -23,11 +23,14 @@ private StudyConstants() {
     public static final String NETWORK_MODIFICATION_API_VERSION = "v1";
     public static final String LOADFLOW_API_VERSION = "v1";
     public static final String SECURITY_ANALYSIS_API_VERSION = "v1";
+    public static final String DYNAMIC_SIMULATION_API_VERSION = "v1";
     public static final String SENSITIVITY_ANALYSIS_API_VERSION = "v1";
     public static final String ACTIONS_API_VERSION = "v1";
     public static final String NETWORK_MAP_API_VERSION = "v1";
     public static final String REPORT_API_VERSION = "v1";
     public static final String SHORT_CIRCUIT_API_VERSION = "v1";
+    public static final String TIME_SERIES_API_VERSION = "v1";
+    public static final String DYNAMIC_MAPPING_API_VERSION = "";
```

Comment:
Add a comment here

Reply:
The dynamic-mapping-server is now without version, must be 'v1' in the next time.
This constant will be fixed when the fix in the dynamic-mapping-server done.
Comment is added